packages/beagle-react/src/components/BeagleDynamicLists/BeagleGridView/index.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+  * Copyright 2020 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *  http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+import React, { FC } from 'react'
+import DynamicListCoreComponent from '../DynamicListCore'
+import { BeagleGridViewInterface, ListType } from '../../../../../common/models'
+
+const BeagleGridView: FC<BeagleGridViewInterface> = (props) => {
+  const gridProps = {
+    ...props,
+    listType: 'GRID' as ListType,
```

Comment:
Based on the suggestion of using String Enums (down on this PR), wouldn't be necessary a cast, I suggest:

```suggestion
    listType: ListType.Grid,
```